README.md

```diff
@@ -93,6 +93,11 @@ iTerm2 color scheme
 - [seoul256-iTerm2](https://github.com/mikker/seoul256-iTerm2) by
 [Mikkel Malmberg](https://github.com/mikker).
 
+Emacs color theme
+-------------------
```

Comment:
A minor nit. The length of the underline doesn't match that of the title. Can you fix it?